toolbox/4.7.2-1/cfg/org.fcrepo.demo.www.cfg

```diff
@@ -0,0 +1,2 @@
+port=${env:DEMO_WWW_PORT:-8099}
```

Comment:
Need some help understanding what this demo is, its purpose, and how to test it?  Should it be a part of this PR or a separate issue?

Reply:
Will remove it and put it in a separate PR if that becomes necessary